### PR TITLE
Fixed cats-generated code

### PR DESCRIPTION
**Pull request by stevevickers**
This PR is needed because I did not measure the boards whilst constructing this part of the bridge 😉 .

### CODE REVIEW

File: src/sbt-test/sbt-scraml/cats/test [hunk 1]
```diff
@@ -0,0 +1,5 @@
+> compile
+# compile again, should not yield errors
```
Comment (adrobisch) [inline src/sbt-test/sbt-scraml/cats/test:2]: I think it's enough to check the "double compilation" test only once in the `simple` test. It's mainly for verifying the cleanup and caching. Kudos for having meaningful assertions :tada:

File: README.md [hunk 2]
```diff
@@ -16,5 +16,5 @@ there is a [test project](src/sbt-test/sbt-scraml/simple) that you can use as a
 
 ```shell
 sbt scripted # runs all tests in src/sbt-test
-sbt scripted sbt-scraml/simple # runs a specific test
+sbt 'scripted sbt-scraml/simple' # runs a specific test (note use of quotes)
```
Comment (adrobisch) [inline README.md:19]: :+1:

**Review by adrobisch — APPROVED** [review]
:bridge_at_night:

File: src/sbt-test/sbt-scraml/cats/test (hunk 1 above)
Comment (stevevickers) [reply src/sbt-test/sbt-scraml/cats/test:2]: Gotcha.  I'll remove the double compilation from `cats` and merge the branch 👍 .

**Pull request merged**